README.md

```diff
@@ -0,0 +1,11 @@
+# Development
+## WSL dev environment setup
+Requirements:
+```bash
+
+sudo apt-get update
+sudo apt-get install -y g++ pkg-config libx11-dev libasound2-dev libudev-dev
+sudo apt-get install libxcursor-dev
+sudo apt-get install alsa
+sudo apt install libegl1-mesa libegl1
```

Comment:
noice!! ty for this!! nit: can we use `apt` or `apt-get` for all of them? 🤣 maybe we can merge the last 3 `apt install`'s into one line na rin since they're WSL-related

Reply:
oh woops thanks for spotting!